docs/moduleAna3AIMEC.rst

```diff
@@ -56,6 +56,13 @@ To run
 
       python3 runScripts/runAna3AIMEC.py
 
+.. Note:: 
+   In the default configuration, the analysis is performed on the simulation result files located in ``NameOfAvalanche/Outputs/anaMod/peakFiles``, where anaMod is specified in the aimecCfg.ini. There is also the option to directly provide a path to an input directory to the :py:func:`ana3AIMEC.ana3AIMEC.fullAimecAnalysis`. However, the peak field file names need to have a specific format: *A_B_C_D_E.asc*, where:
+   
+   - A - *releaseAreaScenario*: refers to the name of the release shape file 
+   - B - *simType*: refers to null (no entrainment, no resistance) 
+   - C - *simulationID*: needs to be unique for the respective simulation and include the model name 
```

Comment:
This only holds true for ISeeSnow, not generally, right? Because we do not include the model name in our name either...

Reply:
that is correct but I thought it would be great to have to then do analysis - but you are correct it is not required, shall I take it out?